CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ else()
     find_package(Lua)
 endif()
 
-find_package(Boost 1.53.0 COMPONENTS system REQUIRED)
+find_package(Boost 1.55.0 COMPONENTS system REQUIRED)
```

Comment:
I'm not questioning the use of 1.55 in Travis, but this is the version requested by CMake. Using 1.55 should still work even if CMake asks for 1.53.